Build failure in CI resolving wasm-loader

Seen here but seems unrelated to these PRs:
https://github.com/spruceid/didkit/pull/86/checks?check_run_id=1992549199#step:23:186
https://github.com/spruceid/didkit/pull/85/checks?check_run_id=1992417392#step:23:186

Excerpt from CI:

```
cd wasm/loader && npm run build

> didkit-wasm-loader@0.1.0 build /home/runner/work/didkit/didkit/didkit/lib/wasm/loader
> webpack --mode development

asset didkit-loader.min.js 8.49 KiB [emitted] [minimized] (name: didkit-loader) 2 related assets
runtime modules 1.08 KiB 4 modules
cacheable modules 17 KiB
  ./didkit-loader.js 285 bytes [built] [code generated]
  ../pkg/didkit_wasm.js 16.7 KiB [built] [code generated]

ERROR in ../pkg/didkit_wasm.js 372:16-63
Module not found: Error: Can't resolve 'wasm-loader' in '/home/runner/work/didkit/didkit/didkit/lib/wasm/loader'
 @ ./didkit-loader.js 4:0-54 8:21-27 10:8-12 11:18-24 12:9-15
```

--- a/lib/wasm/loader/webpack.config.js
+++ b/lib/wasm/loader/webpack.config.js
@@ -8,14 +8,6 @@ module.exports = {
   optimization: {
     minimize: true,
   },
-  module: {
-    rules: [
-      {
-        test: /\.wasm$/,
-        use: [{ loader: "wasm-loader" }],
-      },
-    ],
-  },
   output: {
     path: path.resolve(__dirname),
     filename: "[name].min.js",

